### ISSUE TITLE
C# -> VB: Variable Same Name As Method

### ISSUE THREAD

**Issue by RussellSmith2** [icsharpcode/CodeConverter#487]
### Input code
```csharp
public bool ShouldPreferExplicitType()
{
  bool shouldPreferExplicitType = false;

  return shouldPreferExplicitType;
}
```

### Erroneous output
```vbnet
Public Function ShouldPreferExplicitType() As Boolean
    Dim shouldPreferExplicitType As Boolean = False
    Return shouldPreferExplicitType
End Function
```

### Expected output
```vbnet
Public Function ShouldPreferExplicitType() As Boolean
    Dim someOtherName As Boolean = False
    Return someOtherName 
End Function
```

### Details
VB doesn't allow creating a variable with the same name as the method it is in.

* Version in use: Current compiled repo

**Comment by GrahamTheCoder** [comment]
[CaseConflictResolver.cs:37](https://github.com/icsharpcode/CodeConverter/blob/master/ICSharpCode.CodeConverter/VB/CaseConflictResolver.cs#L37) is probably the place to deal with this case

**Comment by GrahamTheCoder** [comment]
Fixed in #505
